### ISSUE TITLE
Is there a REST API?

### ISSUE THREAD

**Issue by HardeepAsrani** [usefathom/fathom#120]
Is there any REST API for fathom that can be used to, let's say, build a custom dashboard?

**Comment by NTag** [comment]
The roadmap isn't accessible anymore; is it still on it?

**Comment by eivindml-solvr** [comment]
Yeah, any update? 😊 Would love to build a small iOS/macOS dashboard.

**Comment by mgslone** [comment]
@dannyvankooten Is there still any life to this initiative?
